xbmc/powermanagement/PowerManager.cpp

```diff
@@ -298,6 +304,10 @@ void CPowerManager::OnWake()
   g_weatherManager.Refresh();
 
   CServiceBroker::GetPVRManager().OnWake();
+
+  if(m_hadPlayerOnSleep)
```

Comment:
Does anyone have an codeformatter setup for VS which match the project? And is VS able to apply formatting only to changed lines?
It's a little bit anoying to work like in the 90th.
Bad enoght that I have to use logging statements for my test host because it is to slow to use the debug build. Maybe I will try to configure a debugable release build in future.

Reply:
We have our code guidelines but I don't know if someone already configured a code formatter for VS. I'm using Xcode.